### PR TITLE
support brotli

### PR DESCRIPTION
**Pull request by sullis**
## Related

[brotli4j](https://github.com/hyperxpro/Brotli4j)

### CODE REVIEW

**Comment by sullis** [comment]
cc: @hyperxpro

File: client/pom.xml [hunk 1]
```diff
@@ -102,6 +102,42 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>com.aayushatharva.brotli4j</groupId>
+            <artifactId>brotli4j</artifactId>
+            <scope>test</scope>
```
Comment (hyperxpro) [inline client/pom.xml:108]: Make this optional.

Comment (sullis) [reply client/pom.xml]: Done.

**Comment by sullis** [comment]
ready for review

**Comment by hyperxpro** [comment]
So not add dependency in `dependencyManagement`. Move them under `dependencies` and make them optional here.

**Review by hyperxpro — APPROVED** [review]

**Comment by sullis** [comment]
ready to merge

**Comment by hyperxpro** [comment]
Thanks a lot! :)

**Pull request merged**